### PR TITLE
refactor email to step

### LINKED ISSUE #36: Add Step class and field to the recipe

### PR DESCRIPTION
**Pull request by tharshita**

### CODE REVIEW

File: config/travis/deploy_github_pages.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ cd build/docs/html5
 
 git init
 git config user.name 'Deployment Bot (Travis)'
-git config user.email 'deploy@travis-ci.org'
+git config user.step 'deploy@travis-ci.org'
```
Comment (pekchinsiongbrian) [inline config/travis/deploy_github_pages.sh]: HAHAHA harshi, this one doesn't need to change XD it should still be git config user.email

File: docs/ContactUs.adoc [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 * *Bug reports, Suggestions* : Post in our https://github.com/AY1920S2-CS2103T-T10-1/main/issues[issue tracker] if you noticed bugs or have suggestions on how to improve.
 * *Contributing* : We welcome pull requests. Follow the process described https://github.com/oss-generic/process[here]
-* *Email us* : You can also reach us at the following emails:
+* *Step us* : You can also reach us at the following emails:
```
Comment (pekchinsiongbrian) [inline docs/ContactUs.adoc]: Step us XD

**Review by pekchinsiongbrian — COMMENTED** [review]
Thanks for your work harshi!

File: docs/ContactUs.adoc (hunk 2 above)
Comment (beatricetay) [reply docs/ContactUs.adoc]: hahhahha

**Comment by beatricetay** [comment]
Looks good!

**Pull request merged**